lib/analytics.js

```diff
@@ -151,7 +152,7 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(settings, o
     integration.analytics = self;
     integration.once('ready', ready);
     try {
-      integration.initialize();
+      integration.initialize(self._firstPage);
```

Comment:
Once all the integrations are initialized - do we need to keep `_firstPage` around? Could/should we free up the memory used by  it?